pymod/chemistry/atom.py

```diff
@@ -42,6 +42,20 @@ def __repr__(self):
         return 'Atom< name< %s >, %s  radius< %s >, charge< %s >>' \
             % (self.name, self.position, self.radius, self.charge)
 
+    def euclidian_dist2(self, other: 'Atom') -> float:
```

Comment:
It appears your rename killed my comments on the last version of the PR.  For the previous PR version, I used self-control but I can't help myself for this one... this would be a perfect place to use numpy!

Reply:
This is now using numpy, however I'm not sure whether I should sum the array or leave the result to be element-wise. Thoughts?

Reply:
Probably neither; I think you want to use https://numpy.org/doc/stable/reference/generated/numpy.dot.html

Reply:
I put in a TODO to apply https://numpy.org/doc/stable/reference/generated/numpy.dot.html and we will try to get this done today. Going to accept the review and pull request.